FrontEnd/src/components/authorization/LoginContent.jsx

```diff
@@ -34,28 +33,20 @@ const LoginContent = (props) => {
         message: emailErrorText,
       });
       setEnteredEmail("");
-    } else if (
-      enteredEmail.trim().length === 0 &&
-      enteredPassword.trim().length === 0
-    ) {
-      errorMessage = emptyFieldErrorText;
-      setError({
-        errorType: "empty-fields",
-        message: emptyFieldErrorText,
-      });
     } else if (enteredEmail.trim().length === 0) {
       errorMessage = emptyFieldErrorText;
       setError({
-        errorType: "empty-email-field",
+        errorType: "required",
         message: emptyFieldErrorText,
       });
     } else if (enteredPassword.trim().length === 0) {
       errorMessage = emptyFieldErrorText;
       setError({
-        errorType: "empty-password-field",
+        errorType: "required",
         message: emptyFieldErrorText,
       });
     } else if (
+      // fake validation to show functionality
```

Comment:
Why do we need fake validation?

Reply:
I've added this fake field validation to demonstrate functionality (appropriate error messages) on our previous demo. But this fields should be validated on back-end side. Could you recommend please, how can I handle this validation now on front-end side?